app/controllers/friendships_controller.rb

```diff
@@ -0,0 +1,22 @@
+class FriendshipsController < ApplicationController
+  def create
+    id1 = params[:ids][:id1]
+    id2 = params[:ids][:id2]
+    @friendship = Friendship.new(user_id: id1, friend_id: id2)
+    @friendship.save
+    redirect_to users_path
+  end
+
+  def destroy
+    friendship = Friendship.find(params[:friendship_id])
+    friendship.destroy
+    redirect_to user_path(current_user.id)
+  end
+
+  def update
+    friendship = Friendship.find(params[:friendship_id])
+    p current_user
```

Comment:
Kindly remove it, please.

Reply:
it cannot be removed. it update friends when the user clicks on accepts so i can't remove it